packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,26 @@
+# Best Practices
+
+A flat select allows users to select a single value among multiple options.
+
+The width of each of the options should be the same, we don't want any of the options feel more important than the other.
+
+Ideally, offer a maximum of five options.
+
+Don't use Flat select when the effect is not immediate, consider using dropdown menu, radio buttons or checkboxes instead. Users are expect to have immediate feedback on their actions when using Flat select.
+
+If used as view switcher (tabs), make sure it makes sense hierarchically and that tabs cannot be used.
+
+## Labeling
+- Keep titles short, preferably **under three words**.
+- Write for scannability
```

Comment:
@jlevesquecoveo For flat select you don't necessarily need to use verb, since its not a "button" that calls for an action. Its more of a "toggle" between things, for example in Google drive, you can switch between Grid and List view, and are labeled "Grid view" and "List view"

Reply:
Ah, I see. So, any details on how to make the text scannable if it's not about verb placement in this case? 🤔 If it's under three words, I guess there's not much else you can do 😅

Reply:
@jlevesquecoveo Simple, concise words? Hahahah 😅.. What I mean by that is that the user should be able to go through the list of options quickly and easily without too much processing effort

Reply:
Okay, how about:
```suggestion
- Keep titles short, preferably **under three words**.
- Make option labels consistent and easy to scan.
```